src/components/Root/index.tsx

```diff
@@ -37,6 +37,7 @@ const RootConsumer = (): React.ReactElement | null => {
       try {
         await loadChains()
         if (!isValidChainId(_getChainId())) {
+          // Contracts are instantiated in config middleware upon id change
```

Comment:
The comment is irrelevant here.

Reply:
I'll remove it. I noticed that this code is gone in the xDai renaming PR as well.